preset/preset_cli_default.json

```diff
@@ -33,6 +33,8 @@
                     "AudioSecondaryEncoderMode": true,
                     "AudioTrackSelectionBehavior": "first",
                     "ChapterMarkers": true,
+                    "ImageSequence": false,
```

Comment:
Should these really be part of the preset keys? 

Seems more like something that should be handled at source selection.   

Ask for it up-front and maybe remember the last used framerate.